src/jit/liveness.cpp

```diff
@@ -425,6 +466,10 @@ void Compiler::fgPerBlockLocalVarLiveness()
                     break;
             }
         }
+
+        // In minopts, all stores conservatively affect both GcHeap and ByrefExposed memory
+        byrefStatesMatchGcHeapStates = true;
```

Comment:
No, perhaps the comment was misleading.  I don't expect `byrefStatesMatchGcHeapStates` to be checked in minOpts, but leaving it unset didn't seem right.  I've updated the comment to

```c++
        // In minopts, we don't explicitly build SSA or value-number; GcHeap and
        // ByrefExposed implicitly (conservatively) change state at each instr.
```

Is that better?